examples/heat_equation/include/heat_ida.h

```diff
@@ -186,9 +186,10 @@ class Heat : public SundialsInterface<VEC>, public ParameterAcceptor
   bool use_direct_solver;
   bool use_space_adaptivity;
   double kelly_threshold;
-  unsigned int max_dofs;
+  int max_cells;
```

Comment:
this should be unsigned int. 

if it is an int, then it might be too small too soon...